qa-pipelines/qa-pipeline.yml

```diff
@@ -95,6 +95,7 @@ jobs:
         KUBE_ORGANIZATION: ((organization))
         HA: false
         SCALED_HA: false
+        AutoScaler: true
```

Comment:
Why are we mixing cases here? This should be all caps like the rest of them

Reply:
Because its not an abbreviation. If you really want it, I guess I can make the change, but this looked correct to me.

Reply:
@HartS you want APP_AUTO_SCALER or AUTOSCALER?